packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx

```diff
@@ -107,7 +142,7 @@ const WalletBalanceContainer: React.FC = () => {
     .plus(stakedBalance)
     .plus(claimableRewards)
     .decimalPlaces(2)
-    .toFormat();
+    .toFormat(`${availableBalance}` === "0" ? 0 : 2);
```

Comment:
Wouldn't it be better to compare this value as a number type?

Reply:
Fixed